### ISSUE TITLE
Accessibility: MAS40B: Inspect: 'IsExpandCollapsePatternAvailable' property is FALSE for "Open Editors" Toolbar.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17174]
VS Code Version:1.6.1
OS Version:1607

**Steps to Reproduce:**
1.Open Visual studio code editor.
2. In 'Explorer' pane. Navigate to 'Open Editors' toolbar.
3. Using Inspect, Check its "IsExpandCollapsePatternAvailable" property.

**Expected Result:**
'IsExpandCollapsePatternAvailable' property should be TRUE for "Open Editors" Toolbar.

**Actual Result:**
'IsExpandCollapsePatternAvailable' property is FALSE for "Open Editors" Toolbar.
Narrator Info:- Narrator announces as "Open Editors Section, tool bar".
NVDA Info:- NVDA focus goes to the toolbar but announces nothing.
Note: Issue exists for all such toolbars/menus in the app.

**User Impact:**
Users relying on screen readers will not know if the toolbar is expanded or collapsed.

![toolbar expand collapse false](https://cloud.githubusercontent.com/assets/22956130/21184497/bcee9442-c232-11e6-9778-f81ea9009084.png)

**Comment by isidorn** [comment]
No this works fine for latest NVDA on windows, as I focus the open editors section (or any split view for that matter) the title is nicely pronounced and it is readout if it is expanded or collapsed. On expansion / collapse the new state is nicely read out.

As for the flows in narrator I do not really know -> but since we do not support it I am closing this issue.
@WiproPSAccTest please let me know if you still see some issue with NVDA and vscode insiders and I will gladly reopen

![screen shot 2017-05-09 at 17 33 55](https://cloud.githubusercontent.com/assets/1926584/25859086/ba535538-34dd-11e7-9701-6c5019222312.png)

**Comment by chrisdias** [comment]
NVDA seems to be announcing correctly, but the question is why is IsExpandedCollapsePatternAvailable set to false? Should it be true?

**Comment by isidorn** [comment]
After intensively searching online for this I failed to find a reason why this is property is needed.
@WiproPSAccTest might you know why are you testing this?
@chrisdias can you put me in touch with some accessibility expert where I can educate myself more about this

**Comment by isidorn** [comment]
Moving to august until we define why and what is needed here

**Comment by isidorn** [comment]
ping @WiproPSAccTest @chrisdias to check if there are some updates regarding this property

**Comment by isidorn** [comment]
I can not really fix this issue without more details -> closing
